client/package.json

```diff
@@ -2,7 +2,13 @@
   "name": "client",
   "version": "0.1.0",
   "private": true,
+  "proxy": {
+    "/api/*": {
+      "target": "http://localhost:5000"
+    }
+  },
   "dependencies": {
+    "axios": "^0.17.1",
```

Comment:
also when you work on something make sure you have the latest master, cause on master you are using `superagent` and not `axios` so let's be consistent with what we are using

Reply:
https://github.com/FACN3/easyRep/blob/master/client/package.json#L16 this is your master package.json

Reply:
either get rid of superagent or axios

Reply:
Error handling has been done,

Reply:
where ?

Reply:
whats with axios  and the requests that youre making

Reply:
Error handling has been done on the branch email. I have also removed superagent
@Karyum

Reply:
@hoslack you are also using `superagent` https://github.com/FACN3/easyRep/blob/d0e4795afcc133eabef1f18a487d76717e5f663b/client/src/containers/UploadForm.js here, you don't just need to remove it, you also need to refactor this function to use axios